dpnp/tests/test_sum.py

```diff
@@ -58,11 +58,11 @@ def test_sum(shape, dtype_in, dtype_out, transpose, keepdims, order):
         ):
             # If summation is zero and dtype=numpy.bool is passed to numpy.sum
             # NumPy returns True which is not correct
```

Comment:
I guess the comment needs to be updated also

Reply:
Yes, I had updated the comment but had not saved the file :)